docs/source/distributed.tensor.parallel.rst

```diff
@@ -27,31 +27,20 @@ Tensor Parallelism supports the following parallel styles:
 .. autoclass:: torch.distributed.tensor.parallel.style.ColwiseParallel
   :members:
```

Comment:
We should also make sure ColwiseParallel and RowwiseParallel doc (i.e. __init__ doc) with the input_layouts, output_layouts, etc. appear in the doc

Reply:
I will add in the next PR.